SPECS-EXTENDED/libvpx/libvpx.spec

```diff
@@ -212,10 +214,13 @@ rm -rf %{buildroot}%{_prefix}/src
 
 %ldconfig_scriptlets
 
+%check
+make test
```

Comment:
@mandeepsplaha, you mentioned the tests are insanely long (4+ hours). Can we try adding `-j` and see if they run faster? Or do they have issues if ran in parallel (or just don't support that)?